### PR TITLE
Move releases deployment from internal Jenkins to Travis.

### PR DESCRIPTION
**Pull request by ming13**
After the merge I’ll do the following.

* Add environment variables to Travis settings page. I’ve checked scripts and seems like we are not leaking anything.
* Disable internal Jenkins project.

@artem-zinnatullin, anything else?

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,5 +7,12 @@ services:
 script:
   - bash ci/build.sh
 
+deploy:
+  - provider: script
+    script: PUBLISH=true bash ci/build.sh
```
Comment (artem-zinnatullin) [inline .travis.yml]: no need to specify `bash` here, just do:

```console
script: PUBLISH=true ci/build.sh
```

**Comment by artem-zinnatullin** [comment]
@ming13 it should work, the only caveat I have on mind is that current way of getting git tags in gradle scripts doesn't allow one commit to have multiple tags

That can be fixed by using `TRAVIS_TAG` as version instead of calling `git`

File: .travis.yml (hunk 1 above)
Comment (ming13) [reply .travis.yml]: Just copied it from `script` above. Should we remove it too?

Comment (artem-zinnatullin) [reply .travis.yml]: yep!

Comment (ming13) [reply .travis.yml]: Done.

**Comment by artem-zinnatullin** [comment]
lgtm! Don't forget to add env variables on Travis

**Comment by ming13** [comment]
Ready to roll. The next release should be deployed from Travis.

**Pull request merged**